### ISSUE TITLE
max video sockets increase 

### ISSUE THREAD

**Issue by doucheface1** [microsoftgraph/microsoft-graph-comms-samples#307]
is there a way to record more than 10 video sessions?

if we do a conference of more than 9 participants they dont record

we have set SampleConstants.NumberOfMultiviewSockets to 9 which is the limit

but need to know if this can be set to 250 which is video limit in teams

**Comment by 1fabi0** [comment]
Update your Microsoft.Skype.Bots.Media to the newest version you will be able to receive up to 50 concurrent video streams but remind video decoding completly runs on  CPU

**Comment by doucheface1** [comment]
yes but this doesnt help being out of sync? on a two way video call its out of sync when we take raw audio and video and use ffmpeg to join together.

**Comment by doucheface1** [comment]
sorry answering wrong question here

**Comment by Christian-Abbott** [comment]
Which version of Microsoft.Skype.Bots.Media has the higher number of Concurrent videos? We are currently using 1.19.2.25-Aplpa but if we try to increase the max number of sockets it throws an exception.

**Comment by Christian-Abbott** [comment]
I have updated all nuget packages to the latest version and cannot see any newer version of Microsoft.Skype.Bots.Media above 1.19.0.25-Alpha, can you point me to the change log? Thanks

**Comment by 1fabi0** [comment]
You have to activate preview of nuget packages https://docs.microsoft.com/de-de/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages after update Skype bot Media the change log will open

**Comment by ssulzer** [comment]
Within a single bot call media session, there is still a limit of 10 primary/main video VideoSockets + 1 VideoSocket for screen sharing. The note in the readme change log refers to something else, a process-wide limit across all active bot calls.

**Comment by 1fabi0** [comment]
Then send more instances of a call the limit is per application in the call

**Comment by stefan2410** [comment]
@1fabi0  @ssulzer 
There is not  newest version of Microsoft.Skype.Bots.Media above 1.19.0.25-Alpha, even in prerelease
